doc/source/getting_started/install.rst

```diff
@@ -242,8 +242,11 @@ Package                                                          Minimum support
 
 .. _install.recommended_dependencies:
 
-Recommended dependencies
-~~~~~~~~~~~~~~~~~~~~~~~~
+Performance dependencies (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pandas recommends the following optional dependencies for performance gains. These dependencies can be specifically
+installed with ``pandas[performance]`` (i.e. add as optional_extra to the pandas requirement)
```

Comment:
Echoing @Dr-Irv's comment, also here I find this confusing, since at the top of this document we actually start with mentioning `conda install pandas` before `pip install pandas`. So I would personally remove the `pandas[performance]` here, or at least every time we mention such an extra also point to a dedicated section explaining it?